share/OpenMS/CHEMISTRY/Enzymes_RNA.xml

```diff
@@ -55,6 +59,24 @@
       <ITEM name="ThreePrimeGain" value="" type="string" />
       <ITEM name="FivePrimeGain" value="p" type="string" />
     </NODE>
+    <NODE name="mazF">
+      <ITEM name="Name" value="mazF" type="string" />
+      <ITEM name="RegEx" value="(?&lt;=A,C,A)" type="string" />
```

Comment:
I've been reminded that an important property of mazF is its modification-specificity: It doesn't cut before m6A! (Not sure about Am, m1A, m6Am etc.). So you need to adjust the first entry in the regexp list.

Reply:
Byron just mentioned that before "A(m5C)A" would *not* be a cutting site either. He didn't know about other A-methylations.

Reply:
I've updated the regex as well as the test to handle those two exceptions. I've also included a big warning in the description for the enzyme that we don't consider other methylations.